src/setup/templates/ceph.conf

```diff
@@ -0,0 +1,9 @@
+[global]
+fsid = 1f1370bc-8d7c-4694-9038-c142659e4e45
```

Comment:
Add indentation. The actual ceph.conf looks like this 
```
ssc-vm-g4-rhev4-0595:~>cat /etc/ceph/ceph.conf
# minimal ceph.conf for 831fdfb8-84de-11ec-ba55-566fcce40128
[global]
	fsid = 831fdfb8-84de-11ec-ba55-566fcce40128
	mon_host = [v2:10.230.245.62:3300/0,v1:10.230.245.62:6789/0]
```

Reply:
I think we can have ceph.conf without indentation, like it is define in ceph doc - https://docs.ceph.com/en/latest/install/manual-deployment/

And even if we add indentation, but when we save any key-value in file using conf_store api(Conf.save()), it removes indentation.

Reply:
In template though we should still keep its indentated.

> And even if we add indentation, but when we save any key-value in file using conf_store api(Conf.save()), it removes indentation.

This is more of bug to me. Keep a low priority bug for this

Reply:
Indentation is requirement by INI format. This is useful particularly when you want to extend to multiline values in future like ssh keys.